### PR TITLE
Add post start script runnig

### PR DESCRIPTION
**Pull request by kirillovsky**
#1 implemented

### CODE REVIEW

**Comment by kirillovsky** [comment]
@microconfig, what do you think about my solution

File: mgmt/mgmt-core/src/main/java/deployment/mgmt/process/start/StartCommandImpl.java [hunk 1]
```diff
@@ -23,15 +23,52 @@
...
     private final PreStartStep preStartStep;
+    private final PostStartStep postStartStep;
     private final StartStrategy startStrategy;
 
     @Override
     public void startWithArgs(String[] services, String... args) {
         announce("Starting " + Arrays.toString(services));
 
         List<StartHandle> handlers = prepareStart(services, args);
-        if (hasSystemFlag("fakeStart")) return;
```
Comment (alxmatorin) [inline mgmt/mgmt-core/src/main/java/deployment/mgmt/process/start/StartCommandImpl.java]: if case of of fakeStart postStart actions must not be executed. please revert this change

File: mgmt/mgmt-core/src/main/java/deployment/mgmt/process/start/StartCommandImpl.java [hunk 2]
```diff
@@ -23,15 +23,52 @@
...
 
+        if (!hasSystemFlag("fakeStart")) {
+            doStart(handlers);
+        }
+
+        stream(services).forEach(this::executePostStartAction);
```
Comment (alxmatorin) [inline mgmt/mgmt-core/src/main/java/deployment/mgmt/process/start/StartCommandImpl.java]: post actions for a service shoud be executed after the service start. not after all services' startup. the logic must be invoked after the await() method

**Review by alxmatorin — CHANGES_REQUESTED** [review]

File: mgmt/mgmt-core/src/main/java/deployment/mgmt/process/start/StartCommandImpl.java (hunk 1 above)
Comment (kirillovsky) [reply mgmt/mgmt-core/src/main/java/deployment/mgmt/process/start/StartCommandImpl.java]: fixed in 87dc88c18de9f94ba06edd58f7dd4e3112aa5a38

File: mgmt/mgmt-core/src/main/java/deployment/mgmt/process/start/StartCommandImpl.java (hunk 2 above)
Comment (kirillovsky) [reply mgmt/mgmt-core/src/main/java/deployment/mgmt/process/start/StartCommandImpl.java]: fixed in 87dc88c18de9f94ba06edd58f7dd4e3112aa5a38
